city_app/router.py

```diff
@@ -0,0 +1,34 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from city_app import schemas, crud
+from dependencies import get_db
+
+router = APIRouter()
+
+
+@router.post("/cities/", response_model=schemas.City)
+def create_city(city: schemas.CityCreate, db: Session = Depends(get_db)):
+    return crud.create_city(db=db, city=city)
+
+
+@router.get("/cities/", response_model=list[schemas.City])
+def read_cities(db: Session = Depends(get_db)):
+    return crud.get_cities(db=db)
+
+
+@router.get("/cities/{city_id}", response_model=schemas.City)
```

Comment:
```suggestion
@router.get("/cities/{city_id}/", response_model=schemas.City)
```
The same in other similar cases